### PR TITLE
add support on Loongnix-Server (loong64)

### PR DESCRIPTION
**Pull request by ysicing**
add support on Loongnix-Server (loong64)

Signed-off-by: ysicing <i@ysicing.me>

### CODE REVIEW

File: go.mod
Comment (bradfitz) [inline go.mod:5]: We don't like checking in replace directives. They're too invasive for people who depend on this repo.

Can you upstream your changes to `github.com/u-root/uio` first?

**Review by bradfitz — CHANGES_REQUESTED** [review]
(LGTM after removing the `replace` directive)

File: util/endian/little.go
Comment (bradfitz) [inline util/endian/little.go:5]: This part I've merged for you in https://github.com/tailscale/tailscale/pull/6236

File: go.mod
Comment (ysicing) [reply go.mod:5]: https://github.com/u-root/uio/pull/6 
Wait for this to be merged

**Comment by bradfitz** [comment]
Once #6703 is in, tailscaled builds fine. Then the go.mod and go.sum changes can be removed from this PR. Rebase?

**Review by bradfitz — APPROVED** [review]

**Pull request merged**
